build/buildTenDRA.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -ex
+
+VERSION=$1
+if echo ${VERSION} | grep 'trunk'; then
+    VERSION=trunk-$(date +%Y%m%d)
+    BRANCH=main
+else
+    BRANCH=V${VERSION}
+fi
+
+OUTPUT=/root/tendra-${VERSION}.tar.xz
+S3OUTPUT=""
+if echo $2 | grep s3://; then
+    S3OUTPUT=$2
+else
+    OUTPUT=${2-/root/tendra-${VERSION}.tar.xz}
+fi
+
+PREFIX=$(pwd)/prefix
+DIR=$(pwd)/tendra
+git clone --depth 1 -b ${BRANCH} https://github.com/tendra/tendra.git ${DIR}
+
+# no -j (currently breaks)
+pmake -C ${DIR} TARGETARCH=x32_64
+pmake -C ${DIR} TARGETARCH=x32_64 bootstrap-rebuild
+# todo after this point...
+pmake -C ${DIR} PREFIX=${PREFIX} install
+
+export XZ_DEFAULTS="-T 0"
+tar Jcf ${OUTPUT} --transform "s,^./,./tendra-${VERSION}/," -C ${PREFIX} .
+
+if [[ ! -z "${S3OUTPUT}" ]]; then
+    s3cmd put --rr ${OUTPUT} ${S3OUTPUT}
+fi
```

Comment:
If we can't figure out what to do about the TLS error, we could skip the rebuild, and just run the bootstrapped-by-gcc binaries. That'd be enough to get things on their feet, at least.

Reply:
Sounds good ! Any thoughts on what files I need to package to make an "installable" package?

Reply:
I'd intended each tool to be packaged independently (they each provide their own `make install` targets). I think we can add a convenience for the top-level makefile though. I hadn't really imagined anyone would use that outside of development!